village/src/components/SmurfForm.js

```diff
@@ -13,6 +14,16 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    let smurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height
+    }
```

Comment:
```suggestion
    const {name, age, height} = this.state
```

Destructuring here may save you some time and may boost readability (depending - just seems to be the recommendation of the current React elders LOL.)